libcontainer/cgroups/utils.go

```diff
@@ -223,6 +231,29 @@ func GetInitCgroupDir(subsystem string) (string, error) {
 	return getControllerPath(subsystem, cgroups)
 }
 
+func GetInitCgroupPath(subsystem string) (string, error) {
+	cgroup, err := GetInitCgroup(subsystem)
+	if err != nil {
+		return "", err
+	}
+
+	return getCgroupPathHelper(subsystem, cgroup)
+}
+
+func getCgroupPathHelper(subsystem, cgroup string) (string, error) {
```

Comment:
In this function yes, and maybe leave the other comment to `raw.path()` to specify why we use `GetOwnCgroupPath` instead of `GetInitCgroupPath`, I used to get a lot of people ask me about these cgroup path functions...